yamllint/conf/default.yaml

```diff
@@ -21,6 +21,7 @@ rules:
   comments:
     level: warning
     require-starting-space: true
+    ignore-shebang: true
```

Comment:
In discussion in #116 earlier this year, we concluded the opposite :)

Reply:
Oh! Thanks for reminding this discussion @adamchainz, it was completely out of my head 🙂

We didn't conclude anything, but you're right: it could be on. I guess no one would blame us for that.

Reply:
I'll leave this on then unless I hear anything. :)